src/Model/Selectors.re

```diff
@@ -67,3 +67,23 @@ let getActiveConfigurationValue = (state: State.t, f) => {
     Configuration.getValue(~fileType, f, state.configuration);
   };
 };
+
+let getActiveTerminalIdOpt = (state: State.t) => {
```

Comment:
I don't think it's necessary to add an `Opt` suffix. That convention comes from the standard library where the implied default is to raise an exception. The default ought to be to return an `option` though, and instead specify `Exn` if it raises. And since we don't have the historical baggage of the standard library, I think we should follow the more modern convention.

Reply:
Yes, good point! I'd prefer the default to be the one that is strongly-typed and backed by the type system - to ensure we're handling errors and not just falling through and crashing.

I started a rough draft of a 'style guide' to document some of these guidelines.

Reply:
Nice. Looks great!